frontend/quasar.conf.js

```diff
@@ -86,6 +86,7 @@ module.exports = function (ctx) {
       env: {
         INFURA_ID: JSON.stringify(process.env.INFURA_ID),
         BUILD_ENV: JSON.stringify(process.env.NODE_ENV),
+        VUE_APP_GOAT_COUNTER:  '"https://poap-vote.goatcounter.com/count"',
```

Comment:
I think is better if we set it up as an environmental variable and then use it like this:
`JSON.stringify(process.env.VUE_APP_GOAT_COUNTER)`